Makefile

```
@@ -0,0 +1,47 @@
+
+MAKEFILE_ROOT=$(shell pwd)
+help: ## Display this help menu
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+# install: Ensure that the Argo CD namespace exists, that Argo CD is installed, and that CRDs we are using are applied to the current cluster
+install-argo: ## Ensure that the Argo CD namespace exists, and that CRDs we are using are applied to the current cluster
+	kubectl apply -f $(MAKEFILE_ROOT)/backend/config/crd/bases
+	kubectl apply -f $(MAKEFILE_ROOT)/backend-shared/config/crd/bases/managed-gitops.redhat.com_operations.yaml
+	kubectl create ns argocd || true
+	kubectl apply -f https://raw.githubusercontent.com/argoproj/argo-cd/release-2.2/manifests/crds/application-crd.yaml
+
+# start: start all the components
+# ensure goreman is installed, with 'go install github.com/mattn/goreman@latest'
+start:
```

Add description, so that is visible with the `make help`. e.g. you might want to add `ensure goreman is installed, with 'go install github.com/mattn/goreman@latest'`

```suggestion
start: ## start all the components
```